src/index.js

```diff
@@ -58,13 +59,25 @@ function loader(...args) {
 }
 
 function pitch(remainingRequest, prevRequest, dataInput) {
-  const options = loaderUtils.getOptions(this) || {};
-  const cacheDirectory = options.cacheDirectory || path.resolve('.cache-loader');
+  const loaderOptions = loaderUtils.getOptions(this) || {};
+  const defaultOptions = {
```

Comment:
@evilebottnawi Can't the whole `defaultOptions` block be moved for some reason ? 🙃